packages/native/src/components/ChartCard/index.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useCallback, useMemo } from "react";
+import { useTheme } from "styled-components/native";
+import { Flex, GraphTabs, InfiniteLoader } from "../index";
+import * as Animatable from "react-native-animatable";
```

Comment:
@LFBarreto @cgrellard-ledger Hey guys, just realized that this `react-native-animatable` has **not** been added to the dependencies in the package.json file so it works only by sheer luck. 

Could one of you make a follow up pr to fix this please? 🙇